packages/flutter/lib/src/widgets/form.dart

```diff
@@ -243,12 +243,42 @@ class FormState extends State<Form> {
     return _validate();
   }
 
-  bool _validate() {
+  dynamic _validate({bool isBool = true}) {
```

Comment:
We try to avoid dynamic because it turns off the type checker. Also, switching what type is actually returned based on an argument is not a good idea.

Reply:
(also elsewhere in this PR)